_data/locales/es.yml

```diff
@@ -8,7 +8,7 @@ es:
     - Conferencias
     - Comunidad
   index:
-    lead: Bienvenidos a la Comunidad de desarrolladores de Black Python.
+    lead: Más negro que tu formateador
```

Comment:
Does this translate with familiarity in Spanish/other languages?

Reply:
Is something depends of the context. Is like "negro" or "Negro" similar idea like "black" and "Black"

For Spanish/other we will  translate anyway ?